spec/index.html

```diff
@@ -1516,58 +1516,71 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
       <dt id="JSON-lexical-space">The <a>lexical space</a></dt>
       <dd>is the set of all <a data-lt="string">RDF strings</a> that conform to the
         <a data-cite="RFC8259#section-2">JSON Grammar</a> as described in
-        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of
-        [[RFC8259]].</dd>
+        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of [[RFC8259]],
+        which also conform to the requirements of [[[RFC7493]]] [[RFC7493]];
+        additionally, numbers MUST NOT include values that cannot be expressed
+        as an <a href="http://ieeexplore.ieee.org/servlet/opac?punumber=4610933">IEEE 754-2008</a> double precision number.
```

Comment:
Infinity and NaN are not in the lexical space of JSON, so are not represented as values. Virtually all XSD datatypes have a lexical space that includes representations that cannot be expressed in the value space. The situation is no different here.

Reply:
The lexical space of xsd:double appears to be a superset of any of the lexical spaces discussed so I don't see that requiring that as well helps.  The value space of xsd:double appears to be a superset of the value spaces discussed (after integers were eliminated) so that doesn't help either.